docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,318 @@
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril snapshots & certificates
+
+Mithril certificates are produced by the aggregator from the individual signatures. Mithril certificates' security rests upon a chain of trust ultimately pointing at the Genesis certificate. Mithril aggregator maintains snapshots and certificates (aggregated multi-signatures) to be served to clients
+
+* **availability**: No (Snapshots and certificates are just a fallback, a cardano-node client can always retrieve the infromation from the cardano network itself, albeit much more slowly)
+* **confidentiality**: No
+* **integrity**: No (the snapshots and certificates are assumed to be secure, integrity is inherent to the protocol)
+
+#### Mithril Genesis Verification Key
+
+Mithril genesis verification key is stored in [GitHub](https://github.com/input-output-hk/mithril/blob/main/mithril-infra/configuration/release-mainnet/genesis.vkey) and only modifiable through an approved merged PR
+
+* **availability**: Yes (it's needed to verify a whole chain of certificate)
+* **confidentiality**: No
+* **integrity**: No (integrity is inherent to the protocol)
+
+#### Mithril Genesis Signing Key
+
+The corresponding signing key is stored in IOG's VaultWarden, and used only once, when the genesis certificate is generated. 
+
+* **availability**: No (? The key is not needed unless a re-genesis process is required, but then a new key could be used instead?)
+* **confidentiality**: Yes
+* **integrity**: Yes (?)
+
+## Threat & Mitigations
+
+### Resource exhaustion on Cardano relay
+
+- DoS of a `mithril-signer` running next to a `cardano-node` being a relay
+- Assets at risk: 
+   - [Block diffusion](#block-diffusion)
+   - [Mithril signatures diffusion](#mithril-signatures-diffusion)
+
+### Resource exhaustion on Cardano block producer
+
+* Assets at risk:
+  - [Block production](#block-production)
+
+### Block diffusion
+* Diffusion is ensured through the connection between BPs, local relays, and downstream/upstream relays
+* preventing them to operate can harm the Cardano network 
+* Relay hosts connect the BP to the network, 
+* Starving relay hosts prevents Mithril signatures and key registration to be shared
+* Starving a cardano-node running on a relay host would prevent or delay the diffusion of new blocks thus harming 
+* Compromising relay hosts would be an extreme form of starving resources
+
+### Block production
+* An incorrect mithril-signer could _starve_ the cardano-node of computing resources thus preventing it from producing and diffusing blocks in a timely manner
+* Compromising BP host would harms a BP's SPO economic viability
+
+### Hardening Operating System
+
+[Developers portal](https://developers.cardano.org/docs/operate-a-stake-pool/hardening-server) already provides thorough documentation on how to harden a linux-based host to run cardano-node
```

Comment:
This clearly is a mitigation. Against which threat is this helping?

We are likely missing a threat and maybe even some assets, entry points, things to mention in the system analysis to motivate this mitigation (= its great to close some gaps by following this question)

Reply:
Indeed, the threat should be the "SPO's infrastructure security" for which a mitigation is "Hardening Operating System". WDYT?

Reply:
I moved it to the 'info' block of the section